starter_project/normalize_breton_test.py

```diff
@@ -1,15 +1,77 @@
+"""Tests for evaluating Celtic initial consonant mutations."""
+
 import unittest
-import normalize_breton_lib
+import normalize_breton_lib as norm
 
 class TestStringMethods(unittest.TestCase):
+    """Tests for evaluating initial consonant mutations."""
 
     def test_normalize_breton(self):
-        'Test the output of NormalizeBreton.'
-        test_cases = [(('a--bc', 'a-bc'), ('ccb--a', 'ccb-a'), ('ba--aa', 'ba-aa'))]
-        for test in test_cases:
-          for test_case, expected in test:
-            test_fst = normalize_breton_lib.NormalizeBreton(test_case)
+        'Test the output of normalize_breton.'
+        with open("testdata/bre_normalized_sentences.tsv", "r") as filename:
```

Comment:
can we do "as test_file" instead of generic filename?

Reply:
That makes perfect sense!